modules/release-notes/ref-release-notes-breaking-changes.adoc

```diff
@@ -5,5 +5,20 @@
 This section lists breaking changes in {product} {product-version}.
 
 
-None.
+[id="breaking-change-rhidp-5812"]
+== `scopes` parameter now required for GitLab project deploy token creation
 
+The `scopes` parameter is now mandatory for the `gitlab:projectDeployToken:create` action. A valid value, such as `['read_repository']`, must be provided.
+
+.Additional resources
+* link:https://issues.redhat.com/browse/RHIDP-5812[RHIDP-5812]
+
+[id="breaking-change-rhidp-5568"]
+== `dynamicPlugins.cache.volumeClaimSpec` removed, dynamic plugin storage now ephemeral (RHIDP-5568)
```

Comment:
```suggestion
== `dynamicPlugins.cache.volumeClaimSpec` removed from the Helm Chart, dynamic plugin storage now ephemeral (RHIDP-5568)
```

Since this is specific to the Helm Chart, would it make sense to make it explicit?

Reply:
Done